lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
```

Comment:
Ah, your questions was *how* users know this. There is currently nothing saying that. Perhaps we should change this to an `Array` of `:required_gems` (and set the value `:enabled` inside the `register` method based on whether these gems are available). Then on the the future info page, we can list the gem requirements for each markup?